source/js/foundation/template-js-handler/continuous-scrolling-box.js

```diff
@@ -0,0 +1,112 @@
+import { gsap } from "gsap";
+import { ScrollTrigger } from "gsap/ScrollTrigger";
+
+export default () => {
+  /**
+   * Creates slideup intro animation when entering a new article. If history.state indicates
+   * that it is coming from another article via scrolltrigger, then we animate the white box that is overlaying the article.
+   */
+  function triggerIntroTransition() {
+    if (sessionStorage.contScrolling) {
```

Comment:
we probably want a `const CONTINUE_SCROLLING = "contScrolling";` here, just to cut down on potential typo-related bugs